api/.env

```diff
@@ -1,9 +0,0 @@
-<<<<<<< HEAD
-PORT=3000
-MONGO_URI=
-=======
-PORT=4000
```

Comment:
Remove what you have for this .env, it is duplicating the mongo_uri and its adding port 4000.
I have it set to 3000 and pasted my mongo uri